reporting/reporting/Stored Procedures/DimMonthData.sql

```diff
@@ -0,0 +1,21 @@
+﻿
+
+CREATE PROCEDURE [dbo].[DimMonthData]
```

Comment:
@Sreenitha-Sampath Remove redundant empty lines before and after create statement and also in this whole file.